mne/preprocessing/tests/test_ica.py

```diff
@@ -321,10 +322,15 @@ def test_ica_additional():
     corrmap([ica, ica2], (0, 0), threshold=2, plot=False, show=False)
     assert_true(ica.labels_["blinks"] == ica2.labels_["blinks"])
     assert_true(0 in ica.labels_["blinks"])
-    template = _get_ica_map(ica)[0]
+    # test retrieval of component maps as arrays
+    components = ica.get_components()
+    template = components[0]
+    EvokedArray(components.T, ica.info, tmin=0.).plot_topomap([0])
```

Comment:
I would return it as 

n_channels, n_components

so no need to transpose. It's also more natural if you see it as A in X = AS